zeebe/load-tests/project/pom.xml

```diff
@@ -10,9 +10,9 @@
     <relativePath>../../../parent/pom.xml</relativePath>
   </parent>
 
-  <artifactId>zeebe-benchmark</artifactId>
+  <artifactId>camunda-load-tests</artifactId>
```

Comment:
ℹ️ We will as next move this up, so I already named it camunda, instead of zeebe.